Samples/iOS-SwiftUI/iOS-SwiftUI/SwiftUIApp.swift

```diff
@@ -4,6 +4,8 @@ import SwiftUI
 
 @main
 struct SwiftUIApp: App {
+    @UIApplicationDelegateAdaptor private var appDelegate: MyAppDelegate
```

Comment:
l: I would hate ending up with multiple `MyAppDelegate` in the project, what do you think if we rename it to `SwiftUIAppDelegate`.
In case we need an app delegate for each app.

Reply:
we could probably abstract that into a shared class in the future, exposed from SentrySampleShared lib